modules/dreamview/backend/teleop/teleop.cc

```diff
@@ -119,6 +123,11 @@ void TeleopService::Start() {
       [this](const std::shared_ptr<DaemonServiceRpt> &msg) {
         UpdateOperatorDaemonRpt(msg);
       });
+
+  pad_message_writer_ =
+      node_->CreateWriter<PadMessage>(planning_pad_channel);
+
+
```

Comment:
Please run `./scripts/clang-format.sh` to fix style issues.

Reply:
fixed [here](https://github.com/ApolloAuto/apollo/pull/10228/commits/f45083489676b39d68f3d82468216844c30de7d1)